src/modules/launcher/Wox.Infrastructure/Storage/JsonStorage`1.cs

```diff
@@ -122,18 +126,146 @@ private void BackupOriginFile()
 
         public void Save()
         {
+            lock (_saveLock)
+            {
+                try
+                {
+                    string serialized = JsonSerializer.Serialize(_data, _serializerOptions);
+                    File.WriteAllText(FilePath, serialized);
+
+                    Log.Info($"Saving cached data at <{FilePath}>", GetType());
+                }
+                catch (IOException e)
+                {
+                    Log.Exception($"Error in saving data at <{FilePath}>", e, GetType());
+                }
+            }
+        }
+
+        public void Clear()
+        {
+            if (File.Exists(FilePath))
+            {
+                File.Delete(FilePath);
+                LoadDefault();
+                Log.Info($"Deleting cached data at <{FilePath}>", GetType());
+            }
+        }
+
+        public void SaveInformationFile(T data)
+        {
+            lock (_saveLock)
+            {
+                DefaultFileContent = JsonSerializer.Serialize(data, _informationSerializerOptions);
+                _storageHelper.Close(DefaultFileContent);
+            }
+        }
+
+        public bool CheckVersionMismatch(T actualData, bool versionMismatch = false)
```

Comment:
CheckVersionMismatch is called with the default object every time. So this argument is not needed actually I guess.

Reply:
Done.